plone/memoize/view.py

```diff
@@ -43,8 +45,8 @@ def memogetter(*args, **kwargs):
                 context_id,
                 instance.__class__.__name__,
                 func.__name__,
-                args[1:],
-                frozenset(kwargs.items()),
+                json.dumps(args[1:]),
+                json.dumps(kwargs),
```

Comment:
While this change allows unhashable types it prevents using types which are not JSON serialisable, like `datetime.date`. So this change might break existing code.